load_data.py

```diff
@@ -118,7 +118,7 @@ def load_article(article):
     print 'loading article...'
 
 def load_schema_dir(dirpath):
-    for schema_file in os.listdir(dirpath):
+    for schema_file in sorted(os.listdir(dirpath)):
```

Comment:
>How about glob.glob('*.txt')

@stefanv is your comment about globbing still relevant? would that make things more flexible?